e2e/content-services/document-list/document-list-actions.e2e.ts

```diff
@@ -223,10 +223,17 @@ describe('Document List Component - Actions', () => {
             it('[C260060] Should be able to open a file/folder through double click action - folder', async () => {
                 const folderTwoModel = new FolderModel({ name: 'folderTwo' });
                 const numberOfSubFolders = 3;
-                await contentServicesPage.createAndOpenNewFolder(folderTwoModel.name);
+
+                await contentServicesPage.createNewFolder(folderTwoModel.name);
+                const nodeIdSubFolderTwo = await contentServicesPage.getAttributeValueForElement(folderTwoModel.name, 'Node id');
+                await contentServicesPage.openFolder(folderTwoModel.name);
+
                 for (let i = 0; i < numberOfSubFolders; i++) {
-                    await contentServicesPage.createNewFolder('subFolder' + (i + 1));
+                    await uploadActions.createFolder('subfolder' + (i + 1), nodeIdSubFolderTwo);
```

Comment:
Can we move the creation of these folders in the beforeAll?

Reply:
I didn;'t change the test just the way it was doing , in this way the API are faster. We can refactor the test later